v1/destinations/iterable/data/IterableIdentifyConfig.json

```diff
@@ -2,14 +2,16 @@
   {
     "destKey": "email",
     "sourceKeys": [
-      "context.traits.email"
+      "context.traits.email",
```

Comment:
Need to put top-level traits with higher precedence. The same comment goes for similar changes.

Reply:
changed